<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateArcClothoidArc</class>
 <widget class="QDialog" name="CreateArcClothoidArc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create-Arc-Clothoid-Arc</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="4">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>m    L=</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Just draw one line (= radius). First point of the line is the center of one arc and second point is intersection</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>between clothoid and this arc. Radius 1 at beginning and radius 2 at the end of clothoid (counterclockwise).</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Which radius have you drawn?</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>gon                phi2=</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_R">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>not drawn R=</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="lineEdit_phi2">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_phi1">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QPushButton" name="pushButtonGenerate">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="lineEdit_A">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>phi1=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>If you want to draw clockwise, radius 2 is at beginning and radius 1 at end. Insert A or L but not both.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>m                      A=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLineEdit" name="lineEdit_L">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>gon</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QRadioButton" name="radioButton_line1">
     <property name="text">
      <string>drawn radius 1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QRadioButton" name="radioButton_line2">
     <property name="text">
      <string>drawn radius 2</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
